piptools/repositories/pypi.py

```diff
@@ -71,8 +73,11 @@ def __init__(self, pip_args: List[str], cache_dir: str):
 
         self._options: optparse.Values = options
         self._session = self.command._build_session(options)
+        self._target_python = make_target_python(options)
         self._finder = self.command._build_package_finder(
-            options=options, session=self.session
+            options=options,
+            session=self.session,
+            target_python=self._target_python,
```

Comment:
```suggestion
            target_python=make_target_python(options),
```

Could you elaborate on `self._target_python` usage? I don't see any usage except on tests.

Reply:
The usage was within the package finder, but after further testing the package finder is not able to use the target python in the way I thought.